src/main/java/com/github/mauricioaniche/ck/metric/RFC.java

```diff
@@ -2,50 +2,23 @@
 
 import com.github.mauricioaniche.ck.CKClassResult;
 import com.github.mauricioaniche.ck.CKMethodResult;
+import com.github.mauricioaniche.ck.util.JDTUtils;
 import org.eclipse.jdt.core.dom.*;
-
 import java.util.HashSet;
-import java.util.List;
 
 public class RFC implements CKASTVisitor, ClassLevelMetric, MethodLevelMetric {
-
 	private HashSet<String> methodInvocations = new HashSet<String>();
 
 	public void visit(MethodInvocation node) {
-		IMethodBinding binding = node.resolveMethodBinding();
-		count(node.getName()  + "/" + arguments(node.arguments()), binding);
-	}
-
-	private String arguments(List<?> arguments) {
-		if(arguments==null || arguments.isEmpty()) return "0";
-		return "" + arguments.size();
+		String methodName = JDTUtils.getQualifiedMethodFullName(node);
```

Comment:
if the method is a lambda or something crazy, does the `getQualifiedMethodFullName` work?

Reply:
I will test that.

Reply:
It does not work for lambda functions: I returns non qualified methodNames without the signature, e.g.
Instead of MethodInvocation.Complex1.m4/2[java.lang.String,int] it returns m4/0.

Reply:
I was wrong, there was a mistake in my test lambda function.